transport/UdpEndpoint.cpp

```diff
@@ -52,9 +52,9 @@ UdpEndpoint::UdpEndpoint(jobmanager::JobManager& jobManager,
     RtcePoll& epoll,
     bool isShared)
     : BaseUdpEndpoint("UdpEndpoint", jobManager, maxSessionCount, allocator, localPort, epoll, isShared),
-      _iceListeners(maxSessionCount),
-      _dtlsListeners(maxSessionCount * 8),
-      _iceResponseListeners(maxSessionCount * 4)
+      _iceListeners(maxSessionCount * 2),
+      _dtlsListeners(maxSessionCount * 16),
+      _iceResponseListeners(maxSessionCount * 64)
```

Comment:
Each candidate that has not yet returned a response will cause a pending request, waiting for a response that has to be identified by transaction id. Multiple candidates with up to 8 pending requests per session. We try to remove items from this map as soon as we can rely on source port.